index.ts

```diff
@@ -0,0 +1,91 @@
+import "reflect-metadata";
+import { PrismaClient } from "./src/generated/prisma-client/index";
+import { buildSchema } from "type-graphql";
+import { 
+  JournalEntryCrudResolver,
+  UserCrudResolver,
+  relationResolvers
+} from "./src/generated/type-graphql";
+import express from "express";
+import { ApolloServer } from "apollo-server-express";
+
+export interface ApolloContext {
+  expressContext: {
+    req: express.Request,
+    res: express.Response
+  };
+  prisma: PrismaClient;
+}
+
+const prisma = new PrismaClient();
+
+async function main() {
+  const schema = await buildSchema({
+    resolvers: [
+      JournalEntryCrudResolver,
+      UserCrudResolver,
+      ...relationResolvers
```

Comment:
relationResolvers is spread here to include all relations created between tables